From cb414e0504a71707dec19a63e4e7fa7e19e937da Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Mon, 18 Jul 2011 16:37:27 +0800
Subject: [PATCH 0260/2463] ENGR00153217-1 fsl_devices.h: change
 fsl_mxc_lcd_platform_data

change fsl_mxc_lcd_platform_data.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 include/linux/fsl_devices.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index ba5940d..844dbc8 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -213,6 +213,10 @@ struct fsl_mxc_lcd_platform_data {
 	char *core_reg;
 	char *analog_reg;
 	void (*reset) (void);
+	int (*get_pins) (void);
+	void (*put_pins) (void);
+	void (*enable_pins) (void);
+	void (*disable_pins) (void);
 	int default_ifmt;
 	int ipu_id;
 	int disp_id;
-- 
1.7.7.4

